api/ws/workers/middlewares/handshake.js

```diff
@@ -72,11 +72,17 @@ var middleware = {
 		return function(headers, cb) {
 			z_schema.validate(headers, definitions.WSPeerHeaders, error => {
 				if (error) {
+					var errorDescription;
+					if (error[0].path && error[0].path.length) {
```

Comment:
Maybe
```
errorDescription = `${error[0].message}`;
if (error[0].path && error[0].path.length) {
    errorDescription = `${error[0].path}: ${errorDescription}`;
}
```

Reply:
I like the `var somevar;` declaration convention; we use it in other files as well but I can change it if you don't like it.